opus/application/static_media/js/hash.js

```diff
@@ -155,6 +155,10 @@ var o_hash = {
                         case "order":
                             opus.prefs[slug] = value.split(',');
                             break;
+                        case "startobs" || "cart_startobs":
```

Comment:
better case syntax, vs. using an OR:
case "startobs":
case "cart_startobs":
     doStuffHere;
     break;

Reply:
Just for the record, the original code won’t work at all. The || on two strings will yield true so it was just equivalent to “case true:” which is not at all what you wanted.